erpnext/selling/doctype/sales_order/sales_order.py

```
@@ -2005,3 +2106,89 @@ def get_work_order_items(sales_order, for_raw_material_request=0):
+@frappe.whitelist()
+def make_subcontracting_inward_order(source_name, target_doc=None, save=False, submit=False, notify=False):
+	if not is_so_fully_subcontracted(source_name):
+		target_doc = get_mapped_subcontracting_inward_order(source_name, target_doc)
+
+		if (save or submit) and frappe.has_permission(target_doc.doctype, "create"):
+			target_doc.save()
+
+			if submit and frappe.has_permission(target_doc.doctype, "submit", target_doc):
+				try:
+					target_doc.submit()
+				except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)